packages/ui/src/bounty/components/BountyListItem/components/WithdrawalDetails.tsx

```diff
@@ -1,22 +1,36 @@
-import React from 'react'
+import React, { memo } from 'react'
 
 import { DetailBox } from '@/bounty/components/BountyListItem/components/DetailBox'
-import { TokenValue } from '@/common/components/typography'
+import { TextMedium, TokenValue } from '@/common/components/typography'
 import { MemberStack } from '@/memberships/components/MemberStack'
-import memberMock from '@/mocks/data/raw/members.json'
+import { Member } from '@/memberships/types'
 
-export const WithdrawalDetails = () => {
+interface Props {
+  winners?: Member[]
+  entrants?: Member[]
+}
+
+export const WithdrawalDetails = memo(({ winners, entrants }: Props) => {
   return (
     <>
       <DetailBox title="Winners">
-        <MemberStack members={memberMock} max={5} />
-      </DetailBox>
-      <DetailBox title="Entries">
-        <MemberStack members={memberMock} max={5} />
+        {winners ? (
```

Comment:
I think schema will return empty array when there is no entries, so filter on `BountyDetail.tsx` will return empty array again and because it's always truthy `None` fallback is unreachable.

`winners?.length` should be good alternative. WDYT?